k8s/helmfile/tailscale-operator.yaml

```diff
@@ -0,0 +1,10 @@
+repositories:
+  - name: tailscale
+    url: https://pkgs.tailscale.com/helmcharts
+
+releases:
+  - name: tailscale
+    namespace: tailscale
+    chart: tailscale/tailscale-operator
```

Comment:
In order to be able to limit resource usage (i.e. limits/request on CPU and RAM), we also still need a values file for this chart.